src/components/ActionMenu/ActionMenu.test.js

```diff
@@ -1,10 +1,16 @@
 import React from 'react';
 import { render } from '@testing-library/react';
+import {
+  axe,
+  toHaveNoViolations,
+} from 'jest-axe';
 
 import '../../../test/jest/__mock__';
 
 import { ActionMenu } from './ActionMenu';
 
+expect.extend(toHaveNoViolations);
+
```

Comment:
Can this be added to `setupFilesAfterEnv` in the `jest.config.js` to not duplicate this code in each file?
Read this - https://jestjs.io/docs/configuration#setupfilesafterenv-array
https://stackoverflow.com/questions/58103938/how-to-access-expect-in-jest-setupfiles
https://github.com/nickcolley/jest-axe#usage